graphql/resolvers/exerciseCommentCrud.ts

```diff
@@ -2,6 +2,32 @@ import { MutationAddExerciseCommentArgs } from '..'
 import { Context } from '../../@types/helpers'
 import prisma from '../../prisma'
 
+export const getExerciseComments = async (
+  _parent: void,
+  _args: { exerciseId: number }
+) => {
+  const exerciseId = _args.exerciseId
+  return prisma.exerciseComment.findMany({
+    where: { parentId: null, exerciseId },
+    include: {
+      replies: true
```

Comment:
This will only include the exercise comment replies and not the replies nested *replies* like

```
// Main exercise comment
{
  replies: [
    {
      // ...props -> It won't have this, right?
      replies: []
    }
    // reply2...etc
  ]
}
```

Reply:
I think the answer is no

<img width="269" alt="image" src="https://user-images.githubusercontent.com/35906419/199440155-6080b2bc-1c90-45e6-8b99-47354b4a2f78.png">

Reply:
Right it will retrieve all the 'main' comments and one nested layer of replies below that, then we use the `getChildComments` query to get the nested replies beyond that level.